che-editors.yaml

```diff
@@ -603,6 +603,27 @@ editors:
               path: '/?backgroundColor=434343&wss'
               secure: false
               protocol: http
+            - name: code-redirect-1
```

Comment:
I think `code-redirect` was used for CheCode and `theia-redirect` for theia so maybe here it should be using `intellij or idea-redirect`

Reply:
Yeah, make sense, just didn't notice that fact, that theia has own naming for such endpoints. So, I'm going to update the naming, thanks for pointing on that!